packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -2567,29 +2567,36 @@ Content-Type: application/octet-stream
 --abcde12345--
 ```
 
-### Payload_MultiPart_FormData_jsonArrayParts
+### Payload_MultiPart_FormData_complexWithHttpPart
```

Comment:
Remove legacy test case `jsonArrayParts` since it doesn't follow new guideline of https://github.com/Azure/cadl-ranch/issues/570#issuecomment-2089011502